src/turn.ts

```diff
@@ -0,0 +1,47 @@
+import { default as twilio } from "twilio";
+
+/**
+ * Twilio ice server
+ */
+export interface ITwilioIceServer {

```

Comment:
can we use `RtcIceServer` from the runtime (lib.d.ts - should be included for "free") here instead of defining this new struct?

Reply:
They are actually a bit different. Especially about `url` vs `urls`.

Reply:
ah that makes sense, however we can conform the twilio returned results to `RtcIceServer` with ease, right? That would be preferable in this case imo, as we use `RtcIceServer` everywhere else in the stack.

Reply:
Yeah, that's why I made the `requestTwilioTurnServer` return `RTCIceServer[]`. Actually `ITwilioIceServer` is exported just to be used in our unit tests.